pkg/arvo/ted/gcp-jwt.hoon

```diff
@@ -0,0 +1,27 @@
+/-  spider, settings
+/+  jose, pkcs, strandio
+=,  strand=strand:spider
+=,  dejs:format
+^-  thread:spider
+|=  vase
```

Comment:
preferable to put =vase rather than vase by itself

Reply:
What if I don't want to use the vase at all inside this function? I.e. it should take no arguments?

Reply:
went ahead and changed this.